dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1711,3 +1711,25 @@ def get_section_id_from_table(row_index)
   body_text = @browser.execute_script('return document.body && document.body.textContent;').to_s
   expect(body_text.include?(text)).to eq(negation.nil?)
 end
+
+And(/^I update user "([^"]*)" created_at timestamp$/) do |name|
+  require_rails_env
+  user = User.find_by_email_or_hashed_email(@users[name][:email])
+  user.created_at = 8.days.ago
+  user.save!
+end
+
+And(/^I update user "([^"]*)" last_seen_school_info_interstitial$/) do |name|
+  require_rails_env
+  user = User.find_by_email_or_hashed_email(@users[name][:email])
+  user.last_seen_school_info_interstitial = 8.days.ago
+  user.save!
+end
+
+And(/^I update user "([^"]*)" last_confirmation_date and last_seen_school_info_interstitial$/) do |name|
```

Comment:
These steps don't tell the whole story of what the step is doing.  For example, it's a little surprising that
```
I update user "Sam" last_seen_school_info_interstitial
```
effectively ages the account eight days, but
```
I update user "Sam" last_confirmation_date and last_seen_school_info_interstitial
```
ages it a whole year.

Let's update the names to mention something about the time passing.

Totally optional: I bet we could come up with clearer names, deduplicate some code, and be a little more complete in our simulation of time passing with something like this:

```rb
And /^Eight days pass for user "([^"]*)"$/ do |name|
  pass_time_for_user name, 8.days
end

And /^One year passes for user "([%"]*)"$/ do |name|
  pass_time_for_user name, 1.year
end

def pass_time_for_user(user_name, amount_of_time)
  require_rails_env
  user = User.find_by_email_or_hashed_email(@users[name][:email])
  user.created_at -= amount_of_time
  user.last_seen_school_info_interstitial -= amount_of_time if user.last_seen_school_info_interstitial
  user.save!
  user.user_school_infos.each do |info|
    info.last_confirmation_date -= amount_of_time
    info.save!
  end
end
```